server/sonar-server/src/main/java/org/sonar/server/user/ws/ChangePasswordAction.java

```diff
@@ -32,6 +32,7 @@
 
   private static final String PARAM_LOGIN = "login";
   private static final String PARAM_PASSWORD = "password";
+  private static final String PARAM_PREVIOUS_PASSWORD = "previousPassword";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2068)